src/histogram.rs

```diff
@@ -247,6 +299,14 @@ impl HistogramTimer {
         }
     }
 
+    #[cfg(all(feature="nightly", target_os="linux"))]
+    fn new_coarse(histogram: Histogram) -> HistogramTimer {
```

Comment:
I think `new_coares` is more consistent. https://github.com/pingcap/rust-prometheus/pull/108/files/c3b14a044460aae9e7a1678ce87520c01b9ea022#diff-7818b565c2c7c19f572155f609f651abR295